JDK9環境のwarningsが出ている

#660 でJDK9以上をサポートしたが古い書き方等でwarningが多数出ている．
気持ち悪いので直した方が良い．
緊急ではない．

https://circleci.com/gh/kusumotolab/kGenProg/2044#build-parameters/containers/0
```
#!/bin/bash -eo pipefail
./gradlew test
> Task :compileJava
> Task :processResources
> Task :classes

> Task :compileTestJava
/home/circleci/repo/src/test/java/jp/kusumotolab/kgenprog/project/jdt/InsertTimeoutRuleFieldOperationTest.java:16: warning: [dep-ann] deprecated item is not annotated with @Deprecated
public class InsertTimeoutRuleFieldOperationTest {
       ^
1 warning

> Task :processTestResources NO-SOURCE
> Task :testClasses

> Task :test
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by jp.kusumotolab.kgenprog.project.test.MemoryClassLoaderTest (file:/home/circleci/repo/build/classes/java/test/) to field java.lang.ClassLoader.classes
WARNING: Please consider reporting this to the maintainers of jp.kusumotolab.kgenprog.project.test.MemoryClassLoaderTest
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
...
```

--- a/src/test/java/jp/kusumotolab/kgenprog/project/jdt/InsertTimeoutRuleFieldOperationTest.java
+++ b/src/test/java/jp/kusumotolab/kgenprog/project/jdt/InsertTimeoutRuleFieldOperationTest.java
@@ -12,6 +12,7 @@ import jp.kusumotolab.kgenprog.project.TestSourcePath;
  * @deprecated #507 の修正により不要になったため，本クラスは廃止 & Ignore
  */
 @Ignore
+@Deprecated
 public class InsertTimeoutRuleFieldOperationTest {
 
   @Test

